main.py

```diff
@@ -48,7 +48,11 @@ def shutdown(self):
         log.info('NApp kytos/topology shutting down.')
 
     def _get_link_or_create(self, endpoint_a, endpoint_b):
-        new_link = Link(endpoint_a, endpoint_b)
+        try:
+            new_link = Link(endpoint_a, endpoint_b)
+        except ValueError as err:
```

Comment:
I think this is begging for a more specific exception class. `ValueError` is too broad, maybe something like `EndpointIsNone` or `LinkCreationError`.